src/components/alert/alert.spec.tsx

```diff
@@ -15,7 +15,7 @@ describe('bds-alert', () => {
       <bds-alert>
         <mock:shadow-root>
         <div class="alert__dialog">
-          <div class="alert">
+          <div class="alert alert-standard">
```

Comment:
We use the Block, Element, Modifier methodology (BEM) on our CSS selectors, you should add two hyphens to separate the modifier, like: ```alert--standard``` and ```alert--wide```